module/behaviour/skills/FallingRelax/src/FallingRelax.cpp

```diff
@@ -48,19 +48,13 @@ namespace module::behaviour::skills {
     using utility::input::ServoID;
 
     FallingRelax::FallingRelax(std::unique_ptr<NUClear::Environment> environment)
-        : Reactor(std::move(environment))
-        , id(size_t(this) * size_t(this) - size_t(this))
-        , falling(false)
-        , FALLING_ANGLE(0.0f)
-        , FALLING_ACCELERATION(0.0f)
-        , RECOVERY_ACCELERATION()
-        , PRIORITY(0.0f) {
+        : Reactor(std::move(environment)), id(size_t(this) * size_t(this) - size_t(this)) {
 
         // do a little configurating
         on<Configuration>("FallingRelax.yaml").then([this](const Configuration& config) {
             // Store falling angle as a cosine so we can compare it directly to the z axis value
-            double fallingAngle = config["FALLING_ANGLE"].as<double>();
-            FALLING_ANGLE       = cos(fallingAngle);
+            const float fallingAngle = config["FALLING_ANGLE"].as<float>();
```

Comment:
Should this be a double?

Reply:
probably not. most of the stuff in here is float. I don't think it matters too much tho